core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -1490,7 +1490,7 @@ public Map<String,String> tableIdMap() {
   public Text getMaxRow(String tableName, Authorizations auths, Text startRow,
       boolean startInclusive, Text endRow, boolean endInclusive) throws TableNotFoundException {
     return getMaxRow(tableName, auths,
-        new RowRange(startRow, startInclusive, endRow, endInclusive));
+        RowRange.create(startRow, startInclusive, endRow, endInclusive));
```

Comment:
The equivalent Guava API would call this "range" instead of "create". And they also use "open" and "closed" instead of "inclusive" and its negation. I think it would be good to use the "range" method name here, but keep our use of "inclusive" that matches with the key Range API.

Reply:
Addressed in a527552